commands/waitForUrlToContain.js

```diff
@@ -0,0 +1,81 @@
+var util = require('util'),
+    events = require('events');
+
+var WaitForUrlToContain = function() {
+    events.EventEmitter.call(this);
+    this.startTimer = null;
+    this.cb = null;
+    this.ms = null;
+    this.selector = null;
+    this.protocol = require('nightwatch/lib/api/protocol.js')(this.client);
+};
+
+util.inherits(WaitForUrlToContain, events.EventEmitter);
+
+/**
+ * Waiting for url expected
+ * @param  {[type]} url [url expected]
+ * @param  {[type]} milliseconds [time for close assert]
+ * @param  {[type]} timeout [time verify]
+ * @param  {[type]} messages [message output]
+ * @param  {Function} callback [callback]
+ * @return {[type]} [client]
+ */
+WaitForUrlToContain.prototype.command = function(url, milliseconds, timeout, messages, callback) {
```

Comment:
I think url needs to be changed to string? technically we aren't passing it a url, we are passing it a string, and we are waiting for the URL to contain that string. Thoughts?